frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -176,8 +177,9 @@ Feature: Create Application from git form
 
         @regression @manual
         Scenario Outline: Dotnet Builder image detection for git url "<git_url>": A-06-TC13
-            Given user is at Import from git page
+            Given user is at Import from Git page
              When user enters Git Repo url as "<git_url>"
+              And warning message "Unable to detect import strategy" displays
               And warning message "Unable to detect the Builder Image" displays in Builder section
```

Comment:
```suggestion
              And warning message "Unable to detect the Builder Image" displays on the form
```

Reply:
I am unable to find this warning in the design docs

Reply:
Updated according to high level designs